flowlayout/src/androidTest/java/dev/chrisbanes/accompanist/flowlayout/FlowLayoutTest.kt

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2020 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package dev.chrisbanes.accompanist.flowlayout
+
+import androidx.compose.ui.test.junit4.createAndroidComposeRule
+import androidx.test.filters.LargeTest
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.junit.runners.JUnit4
+
+@LargeTest
+@RunWith(JUnit4::class)
+class FlowLayoutTest {
+    @get:Rule
+    val composeTestRule = createAndroidComposeRule<FlowLayoutTestActivity>()
+
+    /**
+     * Needed due to https://issuetracker.google.com/174839536.
+     * Otherwise this module has no tests when running on API < 23.
+     */
+    @Test
+    fun dummyTest() = Unit
```

Comment:
The old Flow tests are here: https://android.googlesource.com/platform/frameworks/support/+/1521e43a8600e6df28179ccd12ac36d8bae89339/compose/foundation/foundation-layout/src/androidAndroidTest/kotlin/androidx/compose/foundation/layout/FlowTest.kt

Reply:
Ah nice, thanks.